services/catalog/src/simcore_service_catalog/models/services_db.py

```diff
@@ -83,6 +90,12 @@ def _update_json_schema_extra(schema: JsonDict) -> None:
     )
 
 
+def _httpurl_to_str(value: HttpUrl | str | None) -> str | None:
+    if isinstance(value, HttpUrl):
+        return f"{value}"
+    return value
```

Comment:
Q: if not none. you do not need the if case at all

Reply:
None is an accepted value. The issue is that HttpUrl is not accepted by the db.